src/documentation/code-review.md

```diff
@@ -0,0 +1,63 @@
+# Code Review
+
+Code review is an essential part of our project and we decided that code and documentation changes in the form of a pull request are reviewed.
```

Comment:
our project. Code and documentation changes come as pull requests, and all pull requests will undergo review.
```suggestion
Code review is an essential part of our project: Code and documentation changes come in as pull requests, and all pull requests will undergo review.
```